PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+        x = open(r'config.ini', 'w')
+        Config.write(x)
+        messagebox.showwarning('Nie znaleziono konfiguracji oddziału!','Nie znaleziono konfiguracji oddziału.\nPrzywrócono domyślną konfigurację - PL50.\nAby ustawić inny oddział, skorzystaj z menu.')
+
+menubar = Menu(root)
+ktoryoddzial = StringVar()
+wlaczniktooltip = IntVar()
+wlaczniktest = IntVar()
+
+def setnumber():
+    try:
+        Config.read('config.ini')
+        Config.set('Uzytkownik', 'oddzial', str(ktoryoddzial.get()))
+        x = open('config.ini', 'w')
+        Config.write(x)
+        root.title("Świadectwomat v"+str(versionnr)+" - "+slownikoddzialow.get(ktoryoddzial.get()))
+        entryprzygotowal['values'] = listaprzygotowal.get(ktoryoddzial.get())
+        entryprzygotowal.current(0)
+        kimjestes()
+        setzatwierdzil(ktoryoddzial.get())
+    except:
+        pass
+
+
+def settooltip():
+    try:
+        Config.read('config.ini')
+        Config.set('Uzytkownik', 'tooltip', str(wlaczniktooltip.get()))
+        x = open('config.ini', 'w')
+        Config.write(x)
+    except:
+        pass
+
+def setdefaulttest():
+    try:
+        Config.read('config.ini')
+        Config.set('Uzytkownik', 'test', str(wlaczniktest.get()))
+        x = open('config.ini', 'w')
+        Config.write(x)
+    except:
+        pass
+
+glownemenu = Menu(menubar, tearoff=0)
+
+oddzialmenu = Menu(menubar,tearoff=0)
+# oddzialmenu.add_radiobutton(label='02 - Kalisz', variable=ktoryoddzial, value='02', command=setnumber)
+oddzialmenu.add_radiobutton(label='04 - Katowice', variable=ktoryoddzial, value='04', command=setnumber)
+oddzialmenu.add_radiobutton(label='05 - Gdańsk', variable=ktoryoddzial, value='05', command=setnumber)
+oddzialmenu.add_radiobutton(label='07 - Poznań', variable=ktoryoddzial, value='07', command=setnumber)
+oddzialmenu.add_radiobutton(label='15 - Toruń', variable=ktoryoddzial, value='15', command=setnumber)
+oddzialmenu.add_radiobutton(label='18 - Częstochowa', variable=ktoryoddzial, value='18', command=setnumber)
+oddzialmenu.add_radiobutton(label='20 - Olsztyn', variable=ktoryoddzial, value='20', command=setnumber)
+oddzialmenu.add_radiobutton(label='50 - Centrala', variable=ktoryoddzial, value='50', command=setnumber)
+oddzialmenu.add_radiobutton(label='61 - Produkcja OEM', variable=ktoryoddzial, value='61', command=setnumber)
+menubar.add_cascade(label="Plik", menu=glownemenu)
+glownemenu.add_cascade(label='Oddział', menu=oddzialmenu)
+glownemenu.add_separator()
+
+glownemenu.add_checkbutton(label='Podpowiedzi', variable=wlaczniktooltip, command=settooltip)
+glownemenu.add_separator()
+glownemenu.add_checkbutton(label='Domyślnie włączone testowanie ciśń.', variable=wlaczniktest, command=setdefaulttest)
+
+
+glownemenu.add_separator()
+glownemenu.add_command(label="Wyjście", command=root.quit)
+
+root.config(menu=menubar)
+
+listawezy = []
+listaparametrow = ['0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0','0']
```

Comment:
Jeśli dobrze mi się wydaje, to parametry z tej listy nie są ze sobą ściśle powiązane (tzn. nie są to kolejne wartości tej samej zmiennej np. długość węża tylko różnych zmiennych). Wydzieliłbym tablicę je do osobnej klasy, w której każde pole miałoby swoją nazwę i potem posługiwał się obiektami tworzonymi z tej klasy. Będzie od razu wiadomo co to jest `listaparametrow[2]` a co to `listaparametrow[9]`